benchmarks/IO.py

```diff
@@ -8,7 +8,7 @@
 TYPES = ('int64', 'float64')
 
 def time_ak_write_read(N_per_locale, trials, dtype, path, seed):
-    print(">>> arkouda write/read")
+    print(">>> numpy {} write/read".format(dtype))
```

Comment:
This looks like it should be arkouda, rather than numpy...

Reply:
Whoops, good catch -- must have screwed up a `sed` command. Fixed now